CIFAR10/train/davidcpage_resnet9_1v100-ec2_pytorch.json

```diff
@@ -0,0 +1,16 @@
+{
+    "version": "v1.0",
+    "author": "davidcpage",
+    "authorEmail": "david@myrtlesoftware.com",
+    "framework": "pytorch 0.4.0",
+    "codeURL": "https://github.com/davidcpage/cifar10-fast/tree/d31ad8d393dd75147b65f261dbf78670a97e48a8", 
+    "model": "Custom ResNet 9",
+    "hardware": "V100 (AWS p3.2xlarge)",
+    "costPerHour": 3.060,
+    "timestamp": "2018-11-06",
+    "misc": {
```

Comment:
For convenience, can you add optimizer details and hyperparameters (e.g. Nesterov, weight decay, batch size, and learning rate) here? As an example, see this [submission](https://github.com/stanford-futuredata/dawn-bench-entries/blob/ba0cd1d4bd8b1eb3aea5d47e67207cc03afeddf7/ImageNet/train/google_resnet50_tpu_tensorflow111.json#L18).

Reply:
I've added optimizer details and parameters. Thanks for the speedy review!